src/beamlime/stateless_workflow.py

```diff
@@ -33,6 +37,7 @@ def __call__(self, group: JSONGroup) -> WorkflowResult:
         }
 
 
-def provide_stateless_workflow() -> StatelessWorkflow:
-    # TODO implement plugin mechanism here, see #132
-    return DummyWorkflow()
+def provide_stateless_workflow(workflow_name: WorkflowName) -> StatelessWorkflow:
+    stateless_workflows = entry_points(group='beamlime.stateless')
+    Workflow = stateless_workflows[workflow_name].load()
+    return Workflow()
```

Comment:
I think we should not assume that a workflow is an instance of a type, it might just be a function (the protocol defines that it has to be callable). The instantiation should thus be done by the module defining the entry point, not here. So this should change to 
```suggestion
    return stateless_workflows[workflow_name].load()
```
Alongside that change, maybe rename `WorkflowName` -> `Workflow` (as well as the command line arg), since that may lead to confusion related to this?